### PR TITLE
Remove an unnecessary redirector in the MixinEntityRenderDispatcher class

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/mixin/MixinEntityRendererDispatcher.java [hunk 1]
```diff
@@ -23,9 +19,11 @@ public void onConfigure(World world, Camera camera, Entity entity, CallbackInfo
         RenderSettings.preRenderEntities();
     }
 
-    @Redirect(method = "shouldRender", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/entity/EntityRenderer;shouldRender(Lnet/minecraft/entity/Entity;Lnet/minecraft/client/render/Frustum;DDD)Z"))
-    public boolean redirectShouldRender(EntityRenderer<Entity> entityRenderer, Entity entity, Frustum frustum, double x, double y, double z) {
-        return entityRenderer.shouldRender(entity, frustum, x, y, z) && RenderSettings.shouldRenderEntity(entity);
+    @Inject(method = "shouldRender", at = @At("RETURN"), cancellable = true)
+    public void redirectShouldRender(Entity entity, Frustum frustum, double x, double y, double z, CallbackInfoReturnable<Boolean> ci) {
+        if (!RenderSettings.shouldRenderEntity(entity)) {
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/mixin/MixinEntityRendererDispatcher.java]: `RenderSettings.shouldRenderEntity` can be expensive in some cases, so this should check if `ci.getReturnValueZ()` is already false before calling that.

**Review by Earthcomputer — CHANGES_REQUESTED** [review]
Oh lol, thanks :)

File: src/main/java/net/earthcomputer/clientcommands/mixin/MixinEntityRendererDispatcher.java (hunk 1 above)
Comment (haykam821) [reply src/main/java/net/earthcomputer/clientcommands/mixin/MixinEntityRendererDispatcher.java]: Fixed in f40e02d80d8d0291b199f13d4ed9e21f5acf7623

**Review by Earthcomputer — APPROVED** [review]
Thanks!

**Pull request merged**